docs/source/rbac/roles.md

```diff
@@ -1,8 +1,6 @@
-(roles)=
-
```

Comment:
I believe this  is actually required for cross-linking from other documents, and shouldn't be removed.

Reply:
Ah, yes! I was going to ask.

I saw this, and didn't know what the purpose was. Naturally, I took it out...instead of asking for clarification. 🤦🏽‍♀️

In hindsight, that's a really dumb thing to do.